model/SystemoveNastaveni/NastrojeDatabaze.php

```diff
@@ -74,6 +74,13 @@ public function vytvorMysqldump(
         );
     }
 
+    public static function fixDefiners(string $filename): void
+    {
+        $file = file_get_contents($filename);
+        $file = preg_replace('#DEFINER=`(?:[^`]|``)*`@`(?:[^`]|``)*`#', '', $file);
```

Comment:
Tvůj popis problému a řešení https://github.com/gamecon-cz/gamecon/pull/552#issuecomment-2227307293 je velmi dobrý.

Tady mi ovšem chybí, protože regexpy jsou obecně těžko čitelné a rozlousknout co že to vlastně řeší chce chvíli soustředění.

Buďto bych přejmenoval metodu na `removeDefiners`, protože to je to co dělá (?), nebo bych sem dal comment proč to odstraňujeme -> co to řeší.

---

Ideál by samozřejmě bylo poslat i pull request do knihovny https://github.com/ifsnop/mysqldump-php .-)
Ale to já osobně dělám jen když mám dobrý den, jinak jim házím jenom issue.